Source/santad/EventProviders/SNTEndpointSecurityManager.mm

```diff
@@ -249,13 +252,18 @@ - (int)postAction:(santa_action_t)action forMessage:(santa_message_t)sm
   es_respond_result_t ret;
   switch (action) {
     case ACTION_RESPOND_ALLOW:
+    case ACTION_RESPOND_ALLOW_COMPILER:
+    case ACTION_RESPOND_ALLOW_PENDING_TRANSITIVE:
       ret = es_respond_auth_result(self.client, (es_message_t *)sm.es_message,
                                    ES_AUTH_RESULT_ALLOW, true);
       break;
     case ACTION_RESPOND_DENY:
+    case ACTION_RESPOND_TOOLONG:
       ret = es_respond_auth_result(self.client, (es_message_t *)sm.es_message,
                                    ES_AUTH_RESULT_DENY, false);
       break;
+    case ACTION_RESPOND_ACK:
+      return ES_RESPOND_RESULT_SUCCESS;
```

Comment:
Does this work? I thought you had to respond before returning?

Reply:
Yep, it works. We just need to ignore the ACKs:
https://github.com/google/santa/blob/master/Source/santad/SNTExecutionController.m#L115

The real response is still sent here:
https://github.com/google/santa/blob/master/Source/santad/SNTExecutionController.m#L160